doc/tutorials/tutorial_post_selection.rst

```diff
@@ -291,20 +301,22 @@ For example, the above user defined ``log`` function can be rewritten using an e
         else:
             return q
 
-    eng, q = sf.Engine(2)
-    with eng:
+    prog = sf.Program(2)
+
+    with prog.context as q:
         MeasureX      | q[0]
         Xgate(RR(q[0],log)) | q[1]
 
-However, ``RegRefTransform`` allows for more flexibility, by allowing us to define a classical processing function that acts on multiple qubits. For example, we can combine two Homodyne measurement results to form a single complex argument for a displacement gate: 
+However, ``RegRefTransform`` allows for more flexibility, by allowing us to define a classical processing function that acts on multiple qubits. For example, we can combine two Homodyne measurement results to form a single complex argument for a displacement gate:
 
 
 .. code-block:: python3
 
-    eng, q = sf.Engine(3)
-    with eng:
+    prog = sf.Program(3)
+
+    with prog.context as q:
         MeasureX      | q[0]
         MeasureP      | q[1]
         Dgate(RR([q[0],q[1]], lambda q0,q1: q0+1j*q1)) | q[2]
 
-:html:`</div></div>`
\ No newline at end of file
+:html:`</div></div>`
```

Comment:
```suggestion
```

Reply:
seems like this was put in to make spacing consistent between different pages. I'm going to leave it in